### PR TITLE
DP-7622  mayflower for information details page

### PR DESCRIPTION
**Pull request by clair0917**
DP-7622 Mayflower for information details page without JS

## Description
PR adds a template for the information details content type, a pullquote component and a js fullWidth image treatment. 

## Steps to Test
Review the Information Details page here: 
https://clair0917.github.io/mayflower/?p=pages-information-details

inVision designs can be found here: 
https://jira.state.ma.us/browse/DP-7282

### CODE REVIEW

File: styleguide/source/assets/js/modules/imageFill.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+export default function (window,document,$,undefined) {
+
+  const sidebarHeight = $('.sidebar').outerHeight( true ) + 100;
+  // Set height of sidebar with some bottom space.
+  let debounceTimer;
+  console.log(sidebarHeight);
```
Comment (isaacchansky) [inline styleguide/source/assets/js/modules/imageFill.js]: remove console log

File: styleguide/source/assets/scss/03-organisms/_pullquote.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
+.ma__pullquote {
+  position: relative;
+
+  &::before {
+    content: '';
+    width: 15%;
+    height: 30px;
+    background-color: green;
```
Comment (isaacchansky) [inline styleguide/source/assets/scss/03-organisms/_pullquote.scss]: Should this be $c-theme-green?

**Review by isaacchansky — CHANGES_REQUESTED** [review]
Looks good! Just a few comments

File: styleguide/source/assets/js/modules/imageFill.js (hunk 1 above)
Comment (clair0917) [reply styleguide/source/assets/js/modules/imageFill.js]: this log has been removed with the PR for DP-7766

**Review by isaacchansky — APPROVED** [review]

**Pull request merged**